dev-libs/libpipeline/libpipeline-1.5.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="A C library for setting up and running pipelines"
+DESCRIPTION="libpipeline is a C library for setting up and running pipelines \
+of processes, without needing to involve shell command-line parsing which is \
+often error-prone and insecure.  This alleviates programmers of the need to \
+laboriously construct pipelines using lower-level primitives such as fork(2) \
+and execve(2)."
+HOMEPAGE="http://libpipeline.nongnu.org/"
+COPYRIGHT="1994 Markus Armbruster.
+	1989, 1990, 1991, 1992, 2000, 2001, 2002, 2003, 2004, 2005,
+	2006, 2007, 2008, 2009, 2010 Free Software Foundation, Inc.
+	2003-2017 Colin Watson"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://download.savannah.nongnu.org/releases/libpipeline/libpipeline-$portVersion.tar.gz"
+CHECKSUM_SHA256="0d72e12e4f2afff67fd7b9df0a24d7ba42b5a7c9211ac5b3dcccc5cd8b286f2b"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libpipeline$secondaryArchSuffix = $portVersion
+	lib:libpipeline$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libpipeline${secondaryArchSuffix}_devel = $portVersion
+	devel:libpipeline$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libpipeline$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libpipeline$secondaryArchSuffix \
+	$libDir/libpipeline.so.$portVersion
```

Comment:
```diff
 defineDebugInfoPackage libpipeline$secondaryArchSuffix \
-	$libDir/libpipeline.so.$portVersion
+	$libDir/libpipeline.so.$libVersion
```

Reply:
And why $libVersion here and not $libVersionCompat?

Reply:
Because after the very first argument, defineDebugInfoPackage expects filenames. The "compat" stuff is just for the `PROVIDES{,_devel}`.